codes/quantum/qudits/quantum_mds.yml

```diff
@@ -0,0 +1,29 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: quantum_mds
+physical: qudits
+logical: qudits
+
+name: 'Quantum Maximum distance separable (MDS) code'
+introduced: '\cite{arXiv:quant-ph/9703048}'
+
+description: |
+  A \((n,q^k,d)_q\) quantum error correction code is an MDS code if parameters \(n\), \(k\), \(d\), and \(q\) are such that the quantum Singleton bound
+  \begin{align}
+  n-k\ge 2(d-1)
+  \end{align}
+  becomes an equality\cite{doi:10.1201/9781315147901}.
+
+protection: 'Given \(n\) and \(k\), quantum MDS codes have the highest distance possible of all codes and so have the best possible error correction properties.'
+
+notes:
+  - '\([[5,1,3]]\) and \([[n,n-2,2]]\), where \(n\) is even, are the only examples of qubit codes that are also quantum MDS codes.'
+  - 'Quantum MDS codes can be constructed through classical generalized Reed-Solomon code\cite{arXiv:2002.06040}. '
+
+relations:
+  parents:
+    - code_id: qudits_into_qudits
+      detail: ''
```

Comment:
Add cousin for stab_5_1_3, explaining it is an MDS code. Is it smallest qubit MDS code?

Reply:
Anotehr cousin: code_id: mds detail: ''

Reply:
I think 5,1,3 is the smallest. Here is the reason.
The smallest distance that a code can correct a single qubit is d=3, so n-k>= 2(3-1) = 4. So for all codes that satisfies n-k=4, [[5,1,3]] seems already to be the smallest.